doc/Makefile

```diff
@@ -1,28 +1,28 @@
 BASEDIR = .
-CLASSES = $(BASEDIR)/classes/ $(BASEDIR)/../modules/
+CLASSES = "$(BASEDIR)/classes/" "$(BASEDIR)/../modules/"
 OUTPUTDIR = $(BASEDIR)/_build
 TOOLSDIR = $(BASEDIR)/tools
```

Comment:
Should those be quoted too for good measure?

Reply:
This would result in double quotes, e.g. `mkdir -p ""output"/rst"`